src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java

```diff
@@ -47,7 +48,7 @@ public void FBButton_003_LoginButton_Visible() {
   public void FBButton_004_ForcedLoginButton_Visible() {
     WikiBasePageObject base = new WikiBasePageObject(driver);
     SpecialNewFilesPageObject specialPage = base.openSpecialNewFiles(wikiURL);
-    specialPage.verifySpecialPage();
+    specialPage.verifyPageHeader(NEW_FILES_SPECIAL_PAGE_TITLE);
```

Comment:
It should be specialPage.verifyPageHeader(specialPage.getNewFilesSpecialPageTitle());